.github/workflows/on-push.yaml

```diff
@@ -0,0 +1,14 @@
+---
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push-to-pages:
+    needs: pre-ci
+    runs-on: ubuntu-latest
+
+    steps:
+      # TODO push this to pages
```

Comment:
reminder for this

Reply:
Done. Kinda.. It's now setup to publish to the github artifacts. As mentioned in the previous comment, we need to work with @bnfinet to get github pages setup.